src/plugins/vis_types/xy/public/config/get_config.test.ts

```diff
@@ -38,7 +38,31 @@ describe('getConfig', () => {
   it('assigns the correct formatter per y axis', () => {
     const config = getConfig(visData, visParamsWithTwoYAxes);
     expect(config.yAxes.length).toBe(2);
-    expect(config.yAxes[0].ticks?.formatter).toStrictEqual(config.aspects.y[1].formatter);
-    expect(config.yAxes[1].ticks?.formatter).toStrictEqual(config.aspects.y[0].formatter);
+    expect(config.yAxes[0].ticks?.formatter).toStrictEqual(config.aspects.y[0].formatter);
```

Comment:
Can you explain this change? It seems like now another formatter is attached per axis but I can't see why that should happen.

Reply:
I changed the implementation of calculating the yAxes array. So in the old way (we were looping into the valueAxes array to compute it), the y Axes was : `["left-axis", "right-axis"]`
With the current implementation (looping the dimensions.y) the yAxes looks like ["right-axis", "left-axis"].